src/ssl/support.cc

```diff
@@ -485,7 +485,7 @@ Ssl::Initialize(void)
 
     SQUID_OPENSSL_init_ssl();
 
-#if HAVE_OPENSSL_ENGINE_H
+#ifndef OPENSSL_NO_ENGINE
```

Comment:
Please use #if instead. The #ifndef check has problems sometimes, so for consistency and a long-term guarantee of build success we only allow it in header protection checks.

```suggestion
#if !defined(OPENSSL_NO_ENGINE)
```